examples/lines_bars_and_markers/eventplot_demo.py

```diff
@@ -49,7 +44,7 @@
 # use individual values for the parameters this time
 # these values will be used for all data sets (except lineoffsets2, which
 # sets the increment between each data set in this usage)
-colors2 = [[0, 0, 0]]
+colors2 = 'k'
```

Comment:
```suggestion
colors2 = 'black'
```
Is more self-explanatory.